addons/hr/models/hr.py

```diff
@@ -112,6 +112,10 @@ def _default_image(self):
     # private partner
     address_home_id = fields.Many2one(
         'res.partner', 'Private Address', help='Enter here the private address of the employee, not the one linked to your company.')
+    address_home_has_company = fields.Boolean(
```

Comment:
I don't really like this field name. Couldn't we find something nice like is_address_home_(a_)company ?